ingeniamotion/communication.py

```diff
@@ -1069,3 +1099,174 @@ def boot_mode_moco(self, servo: str = DEFAULT_SERVO) -> None:
             )
         except ILError as e:
             self.logger.debug(e)
+
+    def __load_ensemble_fw_canopen(
+        self,
+        net: CanopenNetwork,
+        fw_file: str,
+        slave: CanopenServo,
+        status_callback: Optional[Callable[[str], None]] = None,
+        progress_callback: Optional[Callable[[int], None]] = None,
+        error_enabled_callback: Optional[Callable[[bool], None]] = None,
+    ) -> None:
+        """Load FW to an ensemble of servos through Canopen (in parallel).
```

Comment:
In that case, you would just load the FW of a single lfu or sfu file.

Reply:
How? I only have the .kfu file.

Reply:
Well, that case is not supported. Should it be supported @GullinBustin ?

Reply:
Nowadays we want to avoid this feature because these firmware files are destined for groups of drives that must work together. To load a single FW users can use the other functions, but that only has a sense for advanced users, usually it has no sense and is unrecommended.